Example/Example/ContentView.swift

```diff
@@ -0,0 +1,22 @@
+// Created by Cal Stephens on 9/12/23.
+// Copyright © 2023 Airbnb Inc. All rights reserved.
+
+import SwiftUI
+import Lottie
+
+struct ContentView: View {
```

Comment:
Deleted the separate iOS (UIKit) and macOS (AppKit) example apps in favor of a single cross-platform SwiftUI example app.

When we get ready to merge this we'll want to add CI jobs that test all supported platforms on both Xcode 14 and Xcode 15.